collect_app/src/main/java/org/odk/collect/android/instancemanagement/InstanceListItemView.kt

```diff
@@ -0,0 +1,122 @@
+package org.odk.collect.android.instancemanagement
+
+import android.content.Context
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import org.odk.collect.android.R
+import org.odk.collect.android.utilities.FormsRepositoryProvider
+import org.odk.collect.forms.instances.Instance
+import org.odk.collect.strings.R.string
+import timber.log.Timber
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+object InstanceListItemView {
```

Comment:
Why this is not a custom view but a class that only sets its state?. This is not something we used to do.

Reply:
I should have left a comment about this. The problem is that it's used by a `SimpleCursorAdapter` which does the view inflation itself. I didn't want to have to refactor out the cursor, so this was where I ended up. I guess maybe a better approach would be to make this a `ViewHolder` that takes a view in the constructor. That would be overkill for now, but would plugin straight to a `RecyclerView` when all this gets refactored. What do you think?

Reply:
I think that adding a comment in the class that at some point it should be a custom view as the class name says is ok for now.

Reply:
Done!